Remotion/Web/Core/ExecutionEngine/WxeStep.cs

```diff
@@ -127,14 +127,14 @@ public void SetParentStep (WxeStep parentStep)
 
   /// <summary> Gets the step currently being executed. </summary>
   /// <include file='..\doc\include\ExecutionEngine\WxeStep.xml' path='WxeStep/ExecutingStep/*' />
-  public virtual WxeStep ExecutingStep 
+  public virtual WxeStep? ExecutingStep 
```

Comment:
not null, I think.

Reply:
Arg. Okay, WxeIf is a design bug. I think we can do an InvalidOperationExcpetion in the WxeIf for null. ExecutingStep is never called outside of "Executing". In the future, it would likely be that WxeIf.ExecutingStep returns this if execution has not started. WxeIf's an obsolete concept anyway.